src/styles/card.scss

```diff
@@ -0,0 +1,38 @@
+@import "var.scss";
+
+.card {
+  border: 1px solid #f3f3f3;
```

Comment:
It would be great to create variables for all colors, so you can change them in one place later